src/radha/Assigmnent_1_12_withoutTestNG/FindEmpCountForEachDept_TestCase12.java

```diff
@@ -0,0 +1,35 @@
+package radha.Assigmnent_1_12_withoutTestNG;
+
+import java.util.HashMap;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import base.PredefinedActions;
+
+public class FindEmpCountForEachDept_TestCase12 {
+	
+	public static void main(String[] args) {
+		WebDriver driver = PredefinedActions.start("http://automationbykrishna.com/#");
+		driver.findElement(By.xpath("//a[@id='demotable']")).click();
+		HashMap<String, Integer> deptNames = findEmpInEachDept(driver);
+		System.out.println("Employee count in each dept is: "+deptNames);
+		driver.close();
+	}
+	
+	static HashMap<String, Integer> findEmpInEachDept(WebDriver driver){
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method